web/l10n/en.json

```diff
@@ -84,6 +84,11 @@
     "deviceCommand": "Command",
     "deviceFollow": "Follow",
     "deviceTotalDistance": "Total Distance",
+    "deviceStatus": "Status",
+    "deviceSearchFor": "Search for...",
```

Comment:
Do we really need this message? I think it should be pretty obvious even if the field is empty.